.github/workflows/unit-test-mmv1.yml

```diff
@@ -1,13 +1,33 @@
-name: mmv1-lint-product-yaml
+name: mmv1
 
 permissions: read-all
 
 on:
   pull_request:
     paths:
-      - 'mmv1/products/**'
+      - 'mmv1/**'
 
 jobs:
+  version-guard-check:
```

Comment:
was this move just to consolidate similar checks?

Reply:
correct!